Makefile

```diff
@@ -205,18 +205,39 @@ SDL_LDFLAGS += -lopenal
 endif
 SDL_AUDIO_DRIVERS += openal
 endif
-else
+else # ifneq ($(PKG_CONFIG),)
 SDL_CFLAGS := $(shell $(PKG_CONFIG) --cflags sdl2)
 SDL_LDFLAGS := $(shell $(PKG_CONFIG) --libs sdl2) -lpthread
 
 # Allow OpenAL to be disabled even if the development libraries are available
 ifneq ($(ENABLE_OPENAL),0)
-ifeq ($(shell $(PKG_CONFIG) --exists openal && echo 0),0)
+ifneq ($(shell $(PKG_CONFIG) --exists openal && echo 0),)
 SDL_CFLAGS += $(shell $(PKG_CONFIG) --cflags openal) -DENABLE_OPENAL
 SDL_LDFLAGS += $(shell $(PKG_CONFIG) --libs openal)
 SDL_AUDIO_DRIVERS += openal
 endif
 endif
+
+ifneq ($(shell pkg-config --exists gio-2.0 || echo 0),)
```

Comment:
`pkg-config` -> `$(PKG_CONFIG)` :)
 
Also you are checking for `gio-2.0` but in the block below you include `gio-unix-2.0`.
I think you want to check for existence of `gio-unix-2.0` (depends on `gio-2.0`) instead.

See also: https://docs.gtk.org/gio/ and https://docs.gtk.org/gio-unix/